frontend/src/RunCode.js

```diff
@@ -202,6 +200,7 @@ export const _runCode = wrapAsync(async function runCode({code, source}) {
   });
 
   if (error) {
+    console.warn(error)
```

Comment:
What made you add these? I think this one should be removed because `error` is a very large object at this stage and logging it might redundantly send it to Sentry as a breadcrumb. Not sure if the other one is useful in dev.

Reply:
I added them when I was trying to nail down inconsistencies, but no problem to remove them. Will do